test/convert/convertContext.test.ts

```diff
@@ -417,7 +417,7 @@ describe('Convert Transaction Constructs', () => {
           state.exampleComponent = component;
         });
 
-        const result = await context.nonDecomposition.finalize(nonDecomposed.DEFAULT_DIR, TREE);
+        const result = await context.nonDecomposition.finalize(dirname(nonDecomposed.DEFAULT_DIR), TREE);
```

Comment:
Use `dirname` here to strip `force-app` from the path dir.
![Screen Shot 2022-07-06 at 11 13 10](https://user-images.githubusercontent.com/6853656/177571003-4a40d330-b748-42a8-b206-93da5965109c.png)